qutip/tests/test_mesolve.py

```diff
@@ -920,5 +920,53 @@ def f(t, args):
             msg="evolution with feedback not proceding as expected")
 
 
+def test_non_hermitian_dm():
+    """Test that mesolve works correctly for density matrices that are
+    not Hermitian.
+    See Issue #1460
+    """
+    N = 2
+    a = destroy(N)
+    n = num(N)
+    x = (a + a.dag())/np.sqrt(2)
+    H = a.dag() * a
+
+    rho0 = x*fock_dm(N, 0)
```

Comment:
It doesn't matter _too_ much because this is a test and not full code, but this is a pretty confusing setup.  Why create `n` but not use it?

It's also not _immediately_ clear to me that `rho0` is non-Hermitian - I can tell that it is after a second or so of thought, but maybe a comment above it wouldn't hurt.

Reply:
Thank you for reviewing the code! I changed it in the last commit to be more explicit with a comment and removed the unnecessary `n` variable.